webview-ui/src/context/ExtensionStateContext.tsx

```diff
@@ -287,6 +287,7 @@ export const ExtensionStateContextProvider: React.FC<{ children: React.ReactNode
 		includeDiagnosticMessages: true,
 		maxDiagnosticMessages: 50,
 		openRouterImageApiKey: "",
+		kiloCodeImageApiKey: "",
```

Comment:
Our other `kilocodeDefaultModel` vars seem to use a lowercase `code`, maybe they should match?